s3_sync/s3_sync.py

```
@@ -33,126 +34,156 @@
+    os.environ['AWS_SECRET_ACCESS_KEY'] = config['AWS']['AWS_SECRET_ACCESS_KEY']
+    os.environ['slack_webhook_url'] = config['Slack']['webhook_url']
+
 
 def verify_local_folder_exists(local_video_records_done_folder):
-	"""
+    """
     Check whether the local video records done folder exists
 
     Args:
       local_video_records_done_folder (str): The path of the local video
-      	records done folder.
+        records done folder.
 
     Returns:
       If folder exists return None, and if not, logs error and exit.
-	"""
-	if os.path.exists(local_video_records_done_folder):
-		return
-	else:
-		logger.exception("Local Video Records Done folder not found")
-		sys.exit("[-] Local Video Records Done folder not found")
+    """
+    if os.path.exists(local_video_records_done_folder):
+        return
+    else:
+        logger.exception("Local Video Records Done folder not found")
+        sys.exit("[-] Local Video Records Done folder not found")
+
 
 def verify_aws_cli_installed(aws_cli_binary):
-	"""
-	Check whether AWS CLI is installed
+    """
+    Check whether AWS CLI is installed
 
-	Args:
-	  aws cli binary (str): absolute path to aws cli binary file.
+    Args:
+      aws cli binary (str): absolute path to aws cli binary file.
 
-	Returns:
-	  If file exists, return None, else log error and exit.
+    Returns:
+      If file exists, return None, else log error and exit.
+
+    """
+    if os.path.exists(aws_cli_binary):
+        return
+    else:
+        try:
+            r = requests.get(config.get('AWS','CLI', fallback='https://s3.amazonaws.com/aws-cli/AWSCLI64.msi'), stream=True)
+            with open('aws_cli.msi', 'wb') as fd:
+                for chunk in r.iter_content(chunk_size=128):
+                    fd.write(chunk)
+            subprocess.run('msiexec /i aws_cli.msi /qn', check=True,
+                           stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        except:
```

You should almost never use a bare except clause because it can cause maintenance difficulties when an error is captured without you meaning to do so. If you have an understanding of the types of exceptions that may be raised in this context then you should capture those and let the others through.